webapp/public/js/domjudge.js

```diff
@@ -616,67 +613,97 @@ function updateMenuBalloons(data)
             text += data[i].pname + ' ' + data[i].name;
             sendNotification('New balloon:',
                  {'tag': 'ball_' + data[i].baloonid,
-                  'link': domjudge_base_url + '/jury/balloons',
-                  'body': text});
+                        'link': domjudge_base_url + '/jury/balloons',
+                        'body': text});
         }
     }
 }
 
 function initializeAjaxModals()
 {
-	var $body = $('body');
-	$body.on('click', '[data-ajax-modal]', function() {
-		var $elem = $(this);
-		var url = $elem.attr('href');
-		if (!url) {
-			return;
-		}
-		$.ajax({
-			url: url,
-			cache: false
-		}).done(function(data, status, jqXHR) {
-			if (jqXHR.getResponseHeader('X-Login-Page')) {
-				window.location = jqXHR.getResponseHeader('X-Login-Page');
-			} else {
-				var $data = $(data);
-				$('body').append($data);
-				$data.modal('show');
-				if ($elem.data('ajax-modal-after')) {
-					window[$elem.data('ajax-modal-after')]($elem);
-				}
-
-				$data.on('hidden.bs.modal', function () {
-					$data.remove();
-				});
-			}
-		});
-		return false;
-	});
-
-	$body.on('click', '.modal-dialog button[data-url]', function() {
-		var url = $(this).data('url');
-		$.ajax({
-			url: url,
-			method: 'POST'
-		}).done(function(data) {
-			window.location = data.url;
-		});
-	});
+    var $body = $('body');
+    $body.on('click', '[data-ajax-modal]', function() {
+        var $elem = $(this);
+        var url = $elem.attr('href');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mvr320/domjudge/pullRequest?prid=8427924)
```suggestion
		var url = $elem.attr("href");
```